test/e2e/framework/size.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
I don't like the "size.go" name, but I don't have better suggestion, so let's leave :)